internal_metrics/metrics.yaml

```diff
@@ -0,0 +1,253 @@
+  title:  ebpf_net.entrypoint_info
+
+ebpf_net.otlp_grpc.requests_sent:
+  brief: Total number otlp request sent in the prior 30 seconds.
+  description: |
+    The total number of OTLP (Open Telemetry Protocol) request sent for the prior 30 seconds. This is enabled by default.
+  title: ebpf_net.otlp_grpc.requests_sent
+
+ebpf_net.connections:
+  brief: Total number of connections in the prior 30 seconds.
+  description: |
+    The total number of connections for the prior 30 seconds.
+  metric_type: counter
+  title: ebpf_net.connections
+
+ebpf_net.rpc_queue_elem_utilization_fraction:
+  brief: RPC queue utilization fraction in the prior 30 seconds.
+  description: |
+    RPC queue utiliation fraction for the prior thirty seconds.
+  metric_type: counter
+  title: ebpf_net.rpc_queue_elem_utilization_fraction
+
+ebpf_net.disconnects:
+  brief: The total number of disconnects made during the last 30 seconds.
+  description: |
+    The total number of disconnects made during the last 30 seconds.
+  metric_type: counter
+  title: ebpf_net.disconnects
+
+ebpf_net.codetiming_avg_ns:
+  brief: The average code timing time in ns.
+  description: |
+    The average time for a particular instrumented code block in nanoseconds.
+  metric_type: counter
+  title: ebpf_net.codetiming_avg_ns
+
+ebpf_net.client_handle_pool:
+  brief: Client handle pool in the prior 30 seconds.
+  description: |
+    Client handle pool for the prior thirty seconds.
+  metric_type: counte
+  title: ebpf_net.client_handle_pool
+
+ebpf_net.otlp_grpc.successful_requests:
+  brief: Total number of successful OTLP requests in the prior 30 seconds.
+  description: |
+    The total number of successful OTLP requests measured for the prior thirty seconds.
+  metric_type: counter
+  title: ebpf_net.otlp_grpc.successful_requests
+
+ebpf_net.span_utilization:
+  brief: The span unitizaltion in the last 30 seconds.
+  description: |
+    The span unitizaltion in the last 30 seconds.
+  metric_type: counter
+  title: ebpf_net.span_utilization
+
+ebpf_net.up:
+  brief: Total number of apps in the reducer running in the prior 30 seconds.
+  description: |
+    Total number of apps in the reducer running in the prior 30 seconds. This is enabled by default.
+  metric_type: counter
+  title: ebpf_net.up
+
+ebpf_net.rpc_queue_buf_utilization_fraction:
+  brief: RPC queue buffer utilization fraction for the last 30 seconds.
+  description: |
+    RPC queue buffer utilization fraction for the last 30 seconds.
+  title: ebpf_net.rpc_queue_buf_utilization_fraction
+
+ebpf_net.collector_log_count:
+  brief: Ebpf collector log count
+  description: |
+     Ebpf collector log count
+  metric_type: count
+  title: ebpf_net.collector_log_count
+
+ebpf_net.time_since_last_message_ns:
+  brief: Time since last message in ns.
+  description: |
+    Time since last message in ns.
+  metric_type: counter
+  title: ebpf_net.time_since_last_message_ns
+
+ebpf_net.bpf_log:
+  brief: EBPF log count.
```

Comment:
bpf_logs are "error messages from kernel-collector eBPF code that indicate unexpected or unhandled behavior"

Reply:
Thanks @jimwsplk